src/plugins/intel_gpu/src/graph/scatter_elements_update.cpp

```diff
@@ -52,5 +52,30 @@ std::string scatter_elements_update_inst::to_string(scatter_elements_update_node
 }
 
 scatter_elements_update_inst::typed_primitive_inst(network& network, scatter_elements_update_node const& node) : parent(network, node) {}
+void scatter_elements_update_inst::on_execute() {
+    auto input1_shape = _impl_params->input_layouts[1].get_partial_shape();
+    auto input2_shape = _impl_params->input_layouts[2].get_partial_shape();
+
+    if (!input1_shape.is_dynamic() && !input2_shape.is_dynamic()) {
+        if ((input1_shape.get_shape()[0] == 0) || (input2_shape.get_shape()[0] == 0)) {
```

Comment:
What is the shape is like [1, 0]? 
I think we sould check the ov::shape_size(input1_shape.to_shape())

Reply:
It is much better to check empty! I updated it. Thanks!